src/web/services/search.service.ts

```diff
@@ -342,16 +342,25 @@ export class SearchService {
     };
     for (const feedbackSession of feedbackSessions.feedbackSessions) {
       if (this.feedbackSessionService.isFeedbackSessionOpen(feedbackSession)) {
-        feedbackSessionLinks.openSessions[this.feedbackSessionService.generateNameFragment(feedbackSession).toString()]
-          = this.linkService.generateSubmitUrl(student, feedbackSession.feedbackSessionName);
+        feedbackSessionLinks.openSessions[feedbackSession.feedbackSessionName]
+            = {
```

Comment:
It's ok for the `= {` to be on the previous line, i.e.
```ts
feedbackSessionLinks.openSessions[feedbackSession.feedbackSessionName] = {
  // ...
};
```
It saves some levels of indentation.